tests/rustdoc_edge_cases.rs

```diff
@@ -233,9 +233,12 @@ fn release_type_flag_minor() {
         ])
         .assert()
         .failure();
+
+    // Running the same command without the flag should pass.
+    // This is because the detected minimum version change is semver-minor,
```

Comment:
Actually, can we change the `type_marked_deprecated` test crates' versions to 1.0.0 instead, so that this test continues to fail like the comment originally said? I think that's more in the spirit of this test.

If you don't mind, I'd love a parallel test to this one that uses the `function_marked_deprecated` test crates (which are currently also v0.1.0) that ensures the `.success()` case that is the current edit here.